src/main/java/frc/robot/Claw.java

```diff
@@ -0,0 +1,71 @@
+package frc.robot;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.ctre.phoenix.motorcontrol.ControlMode;
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+
+/**
+ * Claw
+ * 
+ * The code for the claw will be posted here for use on the new robot.
+ * 
+ * @author (Charlie Bergdall)
+ * @version (0.0.1) Swenson's Claw uses Neos (Spark MAX), solenoids,
+ */
+public class Claw {
+    //Global Variables
+    public static DoubleSolenoid LPiston, RPiston;
+    public static CANSparkMax claw;
+    public static TalonSRX clawIntakeMotor;
+    
+    //This method should be run first and will initialize all important variables to make the claw work
+    public static void initClaw() {
+        LPiston = new DoubleSolenoid(1, 2); //Creates LPiston | PARAMETERS (forwardChannel, ReverseChannel)
```

Comment:
are these piston ids already used?

Reply:
I won't know what values to put in the parameters until later. Those are placeholders for now.

Reply:
Can you check in the cheezy 2019 repo for the correct id's to use here?

Reply:
Found them as 2 and 5. Should show up in the next commit.

Reply:
updated